lib/rspec/sidekiq/batch.rb

```diff
@@ -33,9 +33,9 @@ def jobs(*)
       class NullStatus < NullObject
         attr_reader :bid
 
-        def initialize(bid, callbacks)
-          @bid = bid
-          @callbacks = callbacks
+        def initialize(bid = nil, callbacks = nil)
```

Comment:
Definitely. Wasn't sure if the `||` pattern in NullBatch#initialize was something I should follow or not. I prefer it your way; I'll make the change. Thanks.